File: app/src/main/java/com/shmedo/mcloudapp/deviceconfig/model/DispatchCmdItem.java
package com.shmedo.mcloudapp.deviceconfig.model;

/**
 * 创建者:   gonghe <br/>
 * 创建时间:  2020/8/30 <br/>
 * 描述：     指令下发/透传接口返回实体类
 */
public class DispatchCmdItem {

    private String deviceToken;//设备SN号
    private String msgID;//消息ID,凭借该ID查询该指令的响应结果

    public String getDeviceToken() {
        return deviceToken;
    }

    public void setDeviceToken(String deviceToken) {
        this.deviceToken = deviceToken;
    }

    public String getMsgID() {
        return msgID;
    }

    public void setMsgID(String msgID) {
        this.msgID = msgID;
    }
}

File: app/src/main/java/com/shmedo/mcloudapp/device/model/DispatchCmdItem.kt
package com.shmedo.mcloudapp.device.model

import com.squareup.moshi.JsonClass

/**
 * 创建者:   gonghe <br></br>
 * 创建时间:  2020/8/30 <br></br>
 * 描述：     指令下发/透传接口返回实体类
 */
@JsonClass(generateAdapter = true)
class DispatchCmdItem(
    val deviceToken: String="", //设备SN号
    val msgID: String="" //消息ID,凭借该ID查询该指令的响应结果
)
